homeassistant/components/sensor/recorder.py

```diff
@@ -33,16 +36,21 @@
     DEVICE_CLASS_BATTERY: {"mean", "min", "max"},
     DEVICE_CLASS_ENERGY: {"sum"},
     DEVICE_CLASS_HUMIDITY: {"mean", "min", "max"},
+    DEVICE_CLASS_MONETARY: {"sum"},
+    DEVICE_CLASS_POWER: {"mean", "min", "max"},
     DEVICE_CLASS_PRESSURE: {"mean", "min", "max"},
     DEVICE_CLASS_TEMPERATURE: {"mean", "min", "max"},
-    DEVICE_CLASS_MONETARY: {"sum"},
 }
 
 UNIT_CONVERSIONS = {
     DEVICE_CLASS_ENERGY: {
         ENERGY_KILO_WATT_HOUR: lambda x: x,
         ENERGY_WATT_HOUR: lambda x: x / 1000,
-    }
+    },
+    DEVICE_CLASS_POWER: {
+        POWER_WATT: lambda x: x,
+        POWER_KILO_WATT: lambda x: x * 1000,
```

Comment:
Why convert kW -> W, when we convert Wh to kWh?

Reply:
W is a more commonly used unit in Home Assistant (not just electricity meters, but bulb, wallplugs and many more). Thus is makes more sense to use that (less conversion).

Reply:
@Danielhiversen Yeah, I agree it's a bit illogical. However, as long as the unit stored is predictable it doesn't matter I think.